sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit.cpp

```diff
@@ -9,8 +9,33 @@
 // REQUIRES: (opencl || level_zero)
 // UNSUPPORTED: accelerator
 
+// UNSUPPORTED: windows
+// UNSUPPORTED-INTENDED: To Do: enable on Windows (via SYCL_ENABLE_EXTENSION_JIT
+// in CMakeLists).
```

Comment:
TODO's are not a valid use of `UNSUPPORTED-INTENDED`. You need to either have a JIRA or a Github issue for the follow-up work.

Reply:
I'm making an inquiry to see if it will be supported in the future and if there is a JIRA. But your sentence confuses me.  

So, if there is a JIRA I just add it to this comment following `UNSUPPORTED-INTENDED: ` correct?  

And if there is not, if there is no intention of ever supporting Windows for this, then what? Just say so in that same comment?

Reply:
> And if there is not, if there is no intention of ever supporting Windows for this, then what? Just say so in that same comment?

Yeah, so something like.

```
UNSUPPORTED-INTENDED: Feature X is not supported on Windows.
```
The more information the better.